app/imports/components/Projects/CreateNewProject/CreateNewProject.jsx

```diff
@@ -1,26 +1,67 @@
 import React, { Component, PropTypes } from 'react';
 import { TextField,
     RaisedButton,
+    AutoComplete,
+    List,
+    ListItem,
 } from 'material-ui';
 
 class CreateNewProject extends Component {
 
     constructor(props) {
         super(props);
         this.onSubmit = this.onSubmit.bind(this);
+        this.addMods = this.addMods.bind(this);
         this.state = {
             isError: false,
             isResult: false,
+            usersNames: [],
+            usersIds: [],
+            chosenModerators: [],
+            chosenMenbers: [],
+            inputError: false,
+            memberError: false,
         };
     }
 
+    componentWillMount() {
+        const users = this.props.userList;
+
+        const names = [];
+        const ids = [];
+        for (let i = 0; i < users.length; i += 1) {
+            names.push(users[i].services.github.username);
```

Comment:
> Profile name is optional

Yes, `profile.name` is optional, but our specification says that user can change his name, it's the best place to keep it.

> first time I`ve used profile.name and got undefinied

During first sign in Meteor fetches data from GitHub. User won't see `undefined`, because server is fast enough to update this data on both sides.

> Also profiles names can be duplicated

Yes, I know it. But this must be clarified.
In my opinion in this case email would be better to identify user.